osde2e/cloud_ingress_operator_tests.go

```diff
@@ -202,9 +208,6 @@ var _ = ginkgo.Describe("cloud-ingress-operator", ginkgo.Ordered, func() {
 
 			ginkgo.By("Initializing AWS ELB service")
 			lb := elb.New(awsSession)
-			input := &elb.DeleteLoadBalancerInput{
-				LoadBalancerName: aws.String(oldLBName),
-			}
```

Comment:
Instead of creating a one-off variable, I moved this into the function call below.